CMakeLists.txt

```diff
@@ -130,7 +130,10 @@ set(ABSL_LOCAL_GOOGLETEST_DIR "/usr/src/googletest" CACHE PATH
   "If ABSL_USE_GOOGLETEST_HEAD is OFF and ABSL_GOOGLETEST_URL is not set, specifies the directory of a local GoogleTest checkout."
   )
 
-if(BUILD_TESTING)
+option(ABSL_BUILD_TESTING
+  "If ON, abweil will build its tests even if abseil is a subproject." OFF)
```

Comment:
Please spell Abseil correctly.

Reply:
The documentation should mention the relationship between this option and BUILD_TESTING.